frontend/src/components/TextField.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "src/components/TextField.module.css";
+
+export interface TextFieldProps extends Omit<React.ComponentProps<"input">, "type" | "className"> {
```

Comment:
can you add a comment explaining the usage of omit? might be confusing for newcomers